renderers/games/teams.py

```diff
@@ -53,10 +53,14 @@ def render_team_banner(canvas, layout, team_colors, home_team, away_team, full_t
                 y_offset = accent_coords[team]["y"]
                 canvas.SetPixel(x + x_offset, y + y_offset, color["r"], color["g"], color["b"])
 
-    __render_team_text(canvas, layout, away_colors, away_team, "away", full_team_names, default_colors)
-    __render_team_text(canvas, layout, home_colors, home_team, "home", full_team_names, default_colors)
+    __render_team_text(canvas, layout, away_colors, away_team, "away", full_team_names, default_colors, use_team_abbrev)
+    __render_team_text(canvas, layout, home_colors, home_team, "home", full_team_names, default_colors, use_team_abbrev)
     __render_team_score(canvas, layout, away_colors, away_team.runs, "away", default_colors)
     __render_team_score(canvas, layout, home_colors, home_team.runs, "home", default_colors)
+    __render_team_hits(canvas, layout, away_colors, away_team.hits, "away", default_colors)
+    __render_team_hits(canvas, layout, home_colors, home_team.hits, "home", default_colors)
+    __render_team_errors(canvas, layout, away_colors, away_team.errors, "away", default_colors)
+    __render_team_errors(canvas, layout, home_colors, home_team.errors, "home", default_colors)
```

Comment:
If it were me, I'd consider doing all of this in part of like a render_team_box method. Doing it separately means it gets weird if any of these numbers hit double digits